less/home-page.less

```diff
@@ -0,0 +1,347 @@
+// home page styles
+.home{
+    .jumbotron{
+      position: relative;
+
+      img{
+        width: 100%;
+        max-height:90vh;
+        z-index:1;
+      }
+
+      h1{
+        .home-heading;
+        z-index:1;
+        position: absolute;
+        bottom: 10%;
+        left: 20%;
+
+        @media @tablet {
+          font-size: 4.5rem;
+          line-height: 60px;
+          left: 10%;
+          
+            @media @phone {
+              font-size: 4rem;
+              line-height: 50px;
+              left: 10%;
+          }
+      }
+
+    }
+
+  }
+
+}
+
+.top-content{
+  display: flex;
+  margin-top: 40px;
+  width: 100%;
+  overflow: hidden;
+
+  .text-content{
+    display: flex;
+    flex-direction: column;
+    align-self: center;
+
+    p{
+      margin-bottom: 20px;
+    }
+
+    h2{
+      @media @tablet {
+        font-size: 2.3rem;
+        margin-top: 20px;
+      }
+      @media @phone {
+        font-size: 2rem;
+        margin-top: 20px;
+      }
+    }
+    
+  }
+
+  @media @tablet {
+    display: flex;
+    flex-direction: column-reverse;
+    
+  }
+  @media @phone {
+    display: flex;
+    flex-direction: column-reverse;
+  }
+
+  .pictures{
+    display: flex;
+    justify-content: center;
+  }
+}
+
+// .middle-content{
+//   display: flex;
+//   margin-top: 100px;
+
+//   .text-content{
+//     display: flex;
+//     flex-direction: column;
+//     align-self: center;
+//     padding-left: 40px;
+
+//     p{
+//       margin-bottom: 20px;
+//     }
+    
+//   }
+//   @media @phone {
+//     display: flex;
+//     flex-direction: column-reverse;
+//     width: 100%;
+//   }
+// }
+
+.middle-content{
+  display: flex;
+  margin-top: 40px;
+  width: 100%;
+  overflow: hidden;
+
+  .text-content{
+    display: flex;
+    flex-direction: column;
+    align-self: center;
+    padding-left: 2%;
+
+    p{
+      margin-bottom: 20px;
+    }
+
+    h2{
+      @media @tablet {
+        font-size: 2.2rem;
+        margin-top: 20px;
+      }
+      @media @phone {
+        font-size: 2rem;
+        margin-top: 20px;
+      }
+    }
+  }
+
+  .pictures{
+    display: flex;
+    justify-content: center;
+  }
+  @media @tablet {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+  @media @phone {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
```

Comment:
Same as with the other instance of this happening. Changes that take place for your tablet media query will also be true for mobile widths. You only need a mobile query if you want to overwrite properties once you hit that 500px breakpoint.